### ISSUE TITLE
可以反向筛选吗

### ISSUE THREAD

**Issue by jdysya** [tohsakrat/flarum-tags-filter#1]
例如我想要排除掉某些标签，是否可行呢？

**Comment by tohsakrat** [comment]
好主意，待我问问后端有没有相应接口，没有就要自己写了

**Comment by therealmaxmoon** [comment]
I agree that it would be much better to deselect tags and start with all, instead of selecting all you want to see.

The reason is, that most people only want to hide one or two tags, instead of click on every tag they want to see.
